src/certified-nodes/communication/email.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Can we create pages from data? I feel like the JSON (or better yet; YAML) should be stored instead of a markdown with frontmatter.

Reply:
You mentioned [here](https://github.com/FlowFuse/website/issues/1257#issuecomment-1804206455) that:

> I think a list and a page per package would be great, though as first iteration a list will suffice.

Is this still the plan? That's why I added all the md files, to have a page per package in place. And I had some questions about it too, [here](https://github.com/FlowFuse/website/pull/1323#issuecomment-1814856726).

Reply:
@Yndira-FlowForge Please read: https://www.11ty.dev/docs/pages-from-data/

Reply:
Thanks for the link :relaxed: I read it and put it into practice, but it only creates html files in the output folders, there's no data that I can use for filtering by category or even display anything from them, and we still would have empty pages. Maybe you have a better idea of how this could be useful that I'm not seeing?

I implemented a different approach that doesn't create any pages, just the list and links to the flows.nodered page. I think as a first iteration it works.

Reply:
@ZJvandeWeg, I believe I understand it now after examining some of the core-nodes files. I'll continue iterating, but I think we should ship it as is for our first iteration. What do you think?